docs/ui/auth/fragments/angular/new-angular-app.md

```diff
@@ -0,0 +1,18 @@
+Use the [Angular CLI](https://github.com/angular/angular-cli) to bootstrap a new Angular app:
+
+```bash
+npm install -g @angular/cli
+ng new myAmplifyProject
+cd myAmplifyProject
+```
+
+### Angular 6+ Support
```

Comment:
I don't think Capi & the front-end rendering of this title (with an ID) will work as expected. Does it need the plus? Can you say "Support for Angular 6 and Above"?

Reply:
Seems to work fine for me

Reply:
Yup, everything seems to be okay when I ran it locally too.

Reply:
Yep––everything's working for me too :) all good